cmd/server/main.go

```diff
@@ -55,14 +57,12 @@ func main() {
 
 	var logger log.Logger
 	if strings.ToLower(*flagLogFormat) == "json" {
-		logger = log.NewJSONLogger(os.Stderr)
+		logger = log.NewJSONLogger()
 	} else {
-		logger = log.NewLogfmtLogger(os.Stderr)
+		logger = log.NewDefaultLogger()
 	}
-	logger = log.With(logger, "ts", log.DefaultTimestampUTC)
-	logger = log.With(logger, "caller", log.DefaultCaller)
 
-	logger.Log("startup", fmt.Sprintf("Starting moov-io/customers server version %s", mainPkg.Version))
+	logger.WithKeyValue("startup", fmt.Sprintf("Starting moov-io/customers server version %s", mainPkg.Version))
```

Comment:
I don't think we should be replacing `logger.Log` with `logger.WithKeyValue` since `logger.WithKeyValue` only adds a `key-value` to the logger's `context` map.

We'd want to use one of these three in `base/log`:
```
        Log(msg string)
	LogError(msg string, err error) error
	LogErrorF(format string, a ...interface{}) error
```

Both `LogError()` and `LogErrorf()` call `Log()` under the hood, they're just convenience methods for returning an error.

Reply:
Do you mean I should:
```go
logger.WithKeyValue("stage?": "startup").Log(fmt.Sprintf("Starting moov-io/customers server version %s", mainPkg.Version))
```
@vxio

Reply:
For me `Log` is the same as `WithKeyValue("msg", "message")` but semantically for sure `Log` has a different meaning.

Reply:
You'd need to call `logger.WithKeyValue().Log()` or `logger.Log()` or `logger.Error()`.

Only calling `WithKeyValue` won't output a log message. It just adds a key-value to the `ctx` map in 
```go
type logger struct {
	writer log.Logger
	ctx    map[string]string
}
```

Reply:
Oh... I got it. Thanks!